network/alsp/internal/reported_misbehavior_work.go

```diff
@@ -0,0 +1,36 @@
+package internal
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/network"
+	"github.com/onflow/flow-go/network/channels"
+)
+
+const NonceSize = 8
+
+// ReportedMisbehaviorWork is an internal data structure for "temporarily" storing misbehavior reports in the queue
+// till they are processed by the worker.
+type ReportedMisbehaviorWork struct {
+	// Channel is the channel that the misbehavior report is about.
+	Channel channels.Channel
+
+	// OriginId is the ID of the peer that the misbehavior report is about.
+	OriginId flow.Identifier
```

Comment:
```suggestion
	NodeId flow.Identifier
```
Lets use NodeId as opposed to OriginId because we are getting the NodeId directly from the identity provider.

Reply:
`OriginId` is consistent with the terminology an [engine is utilizing to attribute a message](https://github.com/onflow/flow-go/blob/master/network/engine.go#L40) to its sender.